doc/tutorial.md

```diff
@@ -27,18 +27,18 @@ One way to use Hedgehog to check the above property is to use the `property` com
 
 ```fs
 property {
-    let! xs = Gen.list (Range.linear 0 100) <| Gen.int (Range.constant 0 1000)
+    let! xs = Gen.list (Range.linear 0 100) (Gen.int (Range.constant 0 1000))
```

Comment:
I think piping right is more readable because then the code is read in the order it is executed.  In this case, I prefer
```fsharp
let! xs = Range.constant 0 1000 |> Gen.int |> Gen.list (Range.linear 0 100)
```

Reply:
Agreed, there are going to be quite a few of those to visit and fix. I'll get to that tomorrow most likely :smile: